pages/services/beta-storage/0.5.3-beta/architecture/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle: Architecture
+title: Architecture
+menuWeight: 20
+excerpt: Architectural overview of storage support in DC/OS
+enterprise: true
+beta: true
+---
+#include /services/include/beta-software-warning.tmpl
+
+The following figure shows the high level architecture of the storage support in DC/OS.
+
+![Storage support in DC/OS](../img/dcos-storage-architecture.png)
```

Comment:
We need to update this graphic.  "Alice" and "Dan" are internal names.  Please replace them with Application Developer and Operator respectively.  Also, please remove the "smiley faces"

Reply:
FYI There's a guide to creating illustrations using Draw.io on the Docs wiki. The Design team has released templates and icons for company-wide use; they make it a lot easier to draw diagrams.

https://wiki.mesosphere.com/display/DOCS/How+to+use+Draw.io+for+technical+illustration